### ISSUE TITLE
hello  when i  npm run dev  i find bug

### ISSUE THREAD

**Issue by yehoan** [nnupoor/webpack4vue#5]
![image](https://user-images.githubusercontent.com/22631753/43436378-d64d6c80-94b6-11e8-89cc-a431d9550bf8.png)

**Comment by yehoan** [comment]
OK i have solve  but i find not can open Browser

**Comment by yehoan** [comment]
hello  why  it happen  this  problem????

**Comment by nnupoor-zz** [comment]
Seems like the versions of vue and vue-template-compiler are not matched. Can you try updating both to latest and try again.
Check version of vue-loader. If your version is >=10.0, update it. Else reinstall it.

**Comment by yehoan** [comment]
yeah  i  find  the vue-loader is 14.1.1  so  i  update  but i find  it no  use  so ..  then  i also  install vue@2.5.13  
npm install vue-template-compiler@2.5.13  i  just  download your profile  and ...  
![image](https://user-images.githubusercontent.com/22631753/43564248-7b1973b0-9658-11e8-8247-d123e96d9c4b.png)

**Comment by yehoan** [comment]
and  also have this  bug ....
